data_processors/pipeline/lambdas/somalier_check.py

```diff
@@ -40,19 +40,24 @@ def handler(event, context) -> dict:
     logger.info(f"Start checking index")
     logger.info(libjson.dumps(event))
 
-    gds_path: str = event['index']
+    index: str = event['index']
 
     # Get name
     timestamp = int(datetime.now().replace(microsecond=0).timestamp())
     step_function_instance_name = "__".join([
         "somalier_check",
-        urlparse(gds_path).path.lstrip("/").replace("/", "_").rstrip(".bam")[-40:],
+        urlparse(index).path.lstrip("/").replace("/", "_").rstrip(".bam")[-40:],
```

Comment:
Sorry to be pedantic, can we put a quick comment on why '-40'?  

i.e before and after of index input variable and whatever the output of `urlparse(index).path.lstrip("/").replace("/", "_").rstrip(".bam")[-40:]` is?

Reply:
Yah. Looks like -40 slice might be broken. I will check again.

Reply:
Added doco with https://github.com/umccr/data-portal-apis/commit/9ec36a2d507dd1318658268424e101fe7cc7e80e